zerver/lib/api_test_helpers.py

```diff
@@ -12,6 +12,7 @@
 
 ZULIP_DIR = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
 FIXTURE_PATH = os.path.join(ZULIP_DIR, 'templates', 'zerver', 'api', 'fixtures.json')
+IMAGE_FIXTURES_DIR = os.path.join(ZULIP_DIR, 'zerver', 'tests', 'fixtures', 'images')
```

Comment:
I think if we're writing tests for this repo, we can just use an existing file under `static/images/integrations`, e.g. a logo or something; I think when Steve suggested adding an additional file, he was thinking about in the python-zulip-api repo, which doesn't have many images in it.  E.g. the GitHub logo can be a custom emoji called "github" for this test.